bin/bash/release-tasks.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
This file has a `.sh` extension but we are using `bash` to run it. Perhaps we should drop the `.sh` extension?